src/contrast.rs

```diff
@@ -17,6 +17,19 @@ use crate::stats::{cumulative_histogram, histogram};
 /// in the (2 * `block_radius` + 1) square block centered on it. If the pixel is at least as bright
 /// as the threshold then it will have a value of 255 in the output image, otherwise 0.
```

Comment:
Perhaps these docs could now just say its equivalent to `adaptive_threshold_with_delta()` with a delta=0 to prevent duplicated docs?